package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "d2",
-  "displayName": "D2",
+  "displayName": "d2",
```

Comment:
shouldn't this remain as `D2`?

Reply:
If that's the only thing I forgot in my signed commit fiasco, I consider myself lucky.